packages/nextjs/app/api/large-milestones/[milestoneId]/review/route.ts

```diff
@@ -89,8 +89,15 @@ export async function POST(req: NextRequest, { params }: { params: { milestoneId
           }
         : {};
 
+    let status = body.status;
```

Comment:
nit: if declare this at the start of the function, multiple `body.status` could be changed to `status`

Reply:
Yes, makes sense. Changed! Thanks!